### ISSUE TITLE
Logging?

### ISSUE THREAD

**Issue by LogicalSoIutions** [atye/ttchat#35]
Possible to add logging so I can ass all the output from a chat?

**Comment by atye** [comment]
Yes, I think that is possible. Do you want the chat in your terminal and in a file or just in a file?

**Comment by atye** [comment]
Do you want to pass in your own file to write to or should `ttchat` create a file tell you where it is?

**Comment by LogicalSoIutions** [comment]
I think ttchat could just ask what dir to make the logs and make the log based off channel name?

**Comment by atye** [comment]
The files can be put in $HOME/.ttchat where config.yaml also is. I think that makes sense.

**Comment by atye** [comment]
Do you want the chat as is or more info like timestamps or whatever else might be possible?

**Comment by LogicalSoIutions** [comment]
I think as is with a time stamp would be good?
